content/reference/admin/private_locations/configuration/ec2/index.md

```diff
@@ -54,6 +54,9 @@ control-plane {
       vpc = "vpc-a"
       # Subnets
       subnets = ["subnet-a", "subnet-b"]
+      # Elastic IP addresses (optional)
+      # You will only be able to deploy a number of load generators corresponding to the number of elastic IP addresses you have configured.
```

Comment:
a number of <concurrent> load generators

corresponding to => up to

elastic IP addresses => Elastic IP addresses